src/main/scala/edu/berkeley/ce/rockslicing/CommandReader.scala

```diff
@@ -0,0 +1,13 @@
+package edu.berkeley.ce.rockslicing
+
+/** A simple tool for parsing command line inputs
+  * @constructor Create a new command reader
+  */
+case class CommandReader(inputs: Array[String]) {
```

Comment:
I know it would require some extra work, but I think we should try to follow standard conventions for our command line arguments. For example, `toVTK` should probably be a switch, so you'd want `-toVTK` or something like that.